lib/cookies/Cookies.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var COOKIES_AVAILABLE_PROPERTY = "brisket:wantsCookies";
+
+var Cookies = {
+
+    letClientKnowIfAvailable: function(environmentConfig, expressRequest) {
+        var cookiesAvailable = typeof expressRequest.cookies === "object";
+
+        environmentConfig[COOKIES_AVAILABLE_PROPERTY] = cookiesAvailable;
+    },
```

Comment:
I don't understand why this is necessary. Why can't the client just read from document.cookie?